docs/source/faq.md

```diff
@@ -101,7 +101,7 @@ UCX takes parameters from specific **environment variables**, which start with t
 prefix `UCX_`.  
 > **IMPORTANT NOTE:** Changing the values of UCX environment variables to non-default
```

Comment:
setting UCX environment variables to non-default values...

Reply:
Done